src/encrypt.rs

```diff
@@ -147,6 +147,7 @@ impl EncryptionKey {
     /// let ct2 = enc_key.rerandomise(ct1, &mut rng);
     /// assert_eq!(dec_key.decrypt(ct1), dec_key.decrypt(ct2));
     /// ```
+    #[must_use = "the Ciphertext input it's not mutated, the function returns the new rerandomised Ciphertext"]
```

Comment:
This is a great idea, thank you very muchh! While I am not maintaining this repository any more, we might as well merge this. Just a small grammar note: please replace `it's` with `is`.